Ronni-ronsbons/week-1/starter-code edited from pt 1/css/main.css

```diff
@@ -0,0 +1,156 @@
+/* Global
+  ======================================================= */
+body {
+  margin: 0;
+  padding: 60px;
+  border-left: 6px solid #CCCCCC;
+  font-family: Lato, sans-serif;
+  line-height: 1.5;
+}
+
+
+/* Layout
+  ======================================================= */
```

Comment:
Good call on both removing the asterisk and specifying _where_ `font-family`, `font-weight` and `width` should go, and partitioning your styling areas with commented sections of responsibility. 👍